can/interfaces/__init__.py

```diff
@@ -31,11 +27,14 @@
     "neousys": ("can.interfaces.neousys", "NeousysBus"),
 }
 
-BACKENDS.update(
-    {
-        interface.name: (interface.module_name, interface.attrs[0])
-        for interface in iter_entry_points("can.interface")
-    }
-)
+try:
+    from importlib.metadata import entry_points
+    entry = entry_points()
+    if 'can.interface' in entry:
+        BACKENDS.update({interface.name: tuple(interface.value.split(':')) for interface in entry['can.interface']})
+except ImportError:
+    from pkg_resources import iter_entry_points as entry_points
+    entry = entry_points("can.interface")
```

Comment:
What drives the need to rename the import here?

Reply:
It was originally leftover code to make things more concise, but this renamed import can be changed if you like.

Reply:
Typically the imports would stay at the top of the module and use a feature flag (like `use_importlib_metadata` for example) to drive this logic here.

Up to you if you want to keep renaming the import or not. Not renaming makes the code easier to read IMO.

Reply:
I'm indifferent, but since you believe the code is easier to read this way, I'll go ahead and make the change for you.